channeldb/channel.go

```diff
@@ -889,6 +889,18 @@ func (c *OpenChannel) String() string {
 	)
 }
 
+// OpeningParty returns the ChannelParty that originally opened this channel.
+func (c *OpenChannel) OpeningParty() lntypes.ChannelParty {
```

Comment:
Lean a little towards `InitiatingParty` (I _think_ it's more commonly used in the codebase) for these, but enough strongly enough to be blocking.

Reply:
I'll level with you. Due to the 80col oppressive regime, I save characters wherever I can.

INITIATING = 10
OPENING = 7

3/80 = 3.75% of the line budget.

Reply:
fairfairfair 😢

Reply:
Yeah I think that works!